packages/cli/generators/project/templates/package.json.ejs

```diff
@@ -52,7 +52,7 @@
   <%_ } -%>
     "migrate": "node ./dist/migrate",
     "openapi-spec": "node ./dist/openapi-spec",
-    "prestart": "npm run build",
+    "prestart": "npm run clean && npm run build",
```

Comment:
pls can you read the following link https://yarnpkg.com/getting-started/migration#explicitly-call-the-pre-and-post-scripts for yarn 2 user

Reply:
Thank you @mdbetancourt. Do we need to check for `yarn` version to correctly adapt the template, in this case ?

[Yarn 1.x documentation](https://classic.yarnpkg.com/en/docs/package-json/) states:

> Certain script names are special. If defined, the preinstall script is called by yarn before your package is installed.

whereas as by [your link](https://yarnpkg.com/getting-started/migration#explicitly-call-the-pre-and-post-scripts), `prestart` script needs to be explicitly called for `yarn 2` users